src/MoBi.Presentation/Mappers/SelectedIndividualToIndividualSelectionDTOMapper.cs

```diff
@@ -18,13 +18,12 @@ public SelectedIndividualToIndividualSelectionDTOMapper(IMoBiContext moBiContext
       {
          _moBiContext = moBiContext;
       }
-      
+
       public IndividualSelectionDTO MapFrom(IndividualBuildingBlock individualBuildingBlock)
       {
-         var individualSelectionDTO = new IndividualSelectionDTO(_moBiContext.CurrentProject.IndividualsCollection)
-         {
-            SelectedIndividualBuildingBlock = individualBuildingBlock
-         };
+         var individualSelectionDTO = new IndividualSelectionDTO(_moBiContext.CurrentProject.IndividualsCollection);
+         if (individualBuildingBlock != null)
+            individualSelectionDTO.SelectedIndividualBuildingBlock = _moBiContext.CurrentProject.IndividualsCollection.FindByName(individualBuildingBlock.Name);
```

Comment:
maybe we need some acessor method . We are always doing so many A.B.C.D . I am not a fan